src/component/Fetchitems.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { FirestoreCollection } from "react-firestore";
+
+// ** NOTES **
+// need to update timestamp settings with DB
+
+const Fetchitems = props => {
```

Comment:
The convention in React is to use [PascalCase](http://wiki.c2.com/?PascalCase) for component names (and their file names) that are composed of 2 or more discrete words. So, in this case, `Fetchitems` should be `FetchItems` both for the component name and the file name.

Consider all of this a soft suggestion. Your stuff works and that's what's important! It is a good habit to get into to use PascalCase, though.